### PR TITLE
Fix indentation in _users.html.erb file 

### PR DESCRIPTION
**Pull request by kirteeprajapati**
Completed my first timer issue i.e fixing indentation in _user.html.erb file
Fixes ##11023 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

image

### CODE REVIEW

**Review by debck — APPROVED** [review]
@kirteeprajapati Great work !

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_users.html.erb [hunk 1]
```diff
@@ -1,32 +1,32 @@
 <script>
 $(document).ready(function () {
-	var table = table_main("#flag_table");
-	$("#all").click(function () { //select all button
-		select_all();
++  var table = table_main("#flag_table");
++  $("#all").click(function () { //select all button
++    select_all();
```
[the remark this answers is not in the document]
Comment (kirteeprajapati) [reply app/views/spam2/_users.html.erb]: Yes i did it is it fine now @TildaDares Kindly review

File: app/views/spam2/_users.html.erb [hunk 2]
```diff
@@ -1,32 +1,32 @@
 <script>
 $(document).ready(function () {
-	var table = table_main("#flag_table");
-	$("#all").click(function () { //select all button
-		select_all();
+   var table = table_main("#flag_table");
```
Comment (TildaDares) [inline app/views/spam2/_users.html.erb:3]:
```suggestion
  var table = table_main("#flag_table");
```

File: app/views/spam2/_users.html.erb [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+   $("#all").click(function () { //select all button
```
Comment (TildaDares) [inline app/views/spam2/_users.html.erb:4]:
```suggestion
  $("#all").click(function () { //select all button
```

File: app/views/spam2/_users.html.erb [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
+     select_all();
```
Comment (TildaDares) [inline app/views/spam2/_users.html.erb:5]:
```suggestion
    select_all();
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Review by TildaDares — APPROVED** [review]
Great work @kirteeprajapati. Thank you!!

**Comment by TildaDares** [comment]
Congratulations @kirteeprajapati 🎉

**Comment by kirteeprajapati** [comment]
> Congratulations @kirteeprajapati 🎉

Thank you so much @TildaDares ✨😊

**Pull request merged**